.github/workflows/unit-test.yml

```
@@ -21,10 +21,10 @@ jobs:
       - name: Checkout code changes
         uses: actions/checkout@v4
 
-      - name: Setup Java 20
+      - name: Setup Java 21
         uses: actions/setup-java@v3
         with:
-          java-version: 20
+          java-version: 21
```

```suggestion
          java-version: 21-ea
```
I'm afraid we need to stick to temurin's early access build until they officially release their 21 stable builds